Lib/test/test_concurrent_futures.py

```diff
@@ -443,6 +451,21 @@ def test_del_shutdown(self):
             p.join()
         call_queue.join_thread()
 
+    def test_shutdown_no_wait(self):
+        executor = futures.ProcessPoolExecutor(max_workers=5)
+        list(executor.map(abs, range(-5, 5)))
```

Comment:
Why the `list` call? Wouldn't you want to test with some pending tasks?

Reply:
Indeed

Reply:
I added a check that make sure all computations are finished when the executor is shut down.
I also updated this in the test just above as it was a mirror of this one.